### PR TITLE
Add UCM2 Configs for Qualcomm sm8550 HDK platform

### PR DESCRIPTION
**Pull request by superna9999**
This adds the UCM2 Configs for Qualcomm SM8550 HDK platform.

Tested with PulseAudio 16.1.

### CODE REVIEW

**Comment by superna9999** [comment]
@perexg Gentle ping!

File: ucm2/codecs/wcd938x/AnalogMic1EnableSeq.conf [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Analog Microphone on Input1 via ADC1
+EnableSequence [
+	cset "name='ADC1_MIXER Switch' 1"
+	cset "name='ADC1 Switch' 1"
+	cset "name='ADC1 Volume' 10"
```
Comment (perexg) [inline ucm2/codecs/wcd938x/AnalogMic1EnableSeq.conf:5]: Volume presets should be only in `BootSequence`. See `Device Volume` section in [this document](https://www.alsa-project.org/alsa-doc/alsa-lib/group__ucm__conf.html).

**Review by perexg — CHANGES_REQUESTED** [review]

File: ucm2/Qualcomm/sm8550/HDK/HiFi.conf [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+		PlaybackPriority 200
+		PlaybackPCM "hw:${CardId},0"
+		PlaybackMixer "default:${CardId}"
+		PlaybackMixerElem "HP Digital"
+		JackControl "Headphone Jack"
+		JackHWMute "Speaker"
+	}
+}
+
+SectionDevice."Top" {
```
[the remark this answers is not in the document]
Comment (superna9999) [reply ucm2/Qualcomm/sm8550/HDK/HiFi.conf]: Ack, fixed thanks

File: ucm2/Qualcomm/sm8550/HDK/HiFi.conf [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+	Include.wcdmice.File "/codecs/wcd938x/AnalogMic5EnableSeq.conf"
+	Include.wcdmicd.File "/codecs/wcd938x/AnalogMic5DisableSeq.conf"
+	# WCD938x ADC4 is routed to TX SWR_INPUT1
+	Include.txmhpe.File "/codecs/qcom-lpass/tx-macro/SoundwireMic1EnableSeq.conf"
+	Include.txmhpd.File "/codecs/qcom-lpass/tx-macro/SoundwireMicDisableSeq.conf"
+
+	Value {
+		CaptureChannels 1
+		CapturePriority 100
```
[the remark this answers is not in the document]
Comment (superna9999) [reply ucm2/Qualcomm/sm8550/HDK/HiFi.conf:95]: Ack, fixed thanks

File: ucm2/codecs/wcd938x/AnalogMic1EnableSeq.conf (hunk 1 above)
Comment (superna9999) [reply ucm2/codecs/wcd938x/AnalogMic1EnableSeq.conf]: Ack, fixed thanks

**Pull request closed without merging**